css/css-values/getComputedStyle-calc-mixed-units-003.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+
+  <meta charset="UTF-8">
+
+  <title>CSS Values Test: computed value of calc() values using multiplication/division with mixed units</title>
+
+  <link rel="help" href="https://www.w3.org/TR/css-values-4/#calc-computed-value">
+
+  <meta name="flags" content="">
+  <meta content="This meta-test checks for the resolution and absolutization of computed values with mixed units to 'px' when multiplication/division is used." name="assert">
+
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+
+  <style>
+  html {
+      font-size: 30px;
+  }
+
+  body {
+      font-size: 16px;
+      line-height: 1.25; /* computed value: 20px */
+      width: 520px;
+      height: 500px;
+      margin: 20px;
+  }
+
+  div#target {
+      height: 100px;
+      width: 100px;
+  }
+  </style>
+
+  <div id="target"></div>
+
+  <script>
+  function startTesting() {
+    var targetElement = document.getElementById("target");
+
+    function verifyComputedStyle(property_name, specified_value, expected_value, description) {
+      test(function() {
+        targetElement.style.setProperty(property_name, "initial");
+        targetElement.style.setProperty(property_name, specified_value);
+        assert_equals(getComputedStyle(targetElement)[property_name], expected_value);
+      }, description);
+    }
+
+    verifyComputedStyle("width", "calc(10% * 10 + 2em)", "552px", "testing width: calc(10% * 10 + 2em)");
+    /*
+      10% of 520px = 52px
+      2em = 32px
+      52px * 10 + 32px = 520px + 32px = 552px
+      Total = 552px
+    */
+
+    verifyComputedStyle("width", "calc(5% * 10 + 4em)", "324px", "testing width: calc(5% * 10 + 4em)");
+    /*
+      5% of 520px = 26px
+      4em = 64px
+      26px * 10 + 64px = 260px + 64px = 324px
+      Total = 324px
+    */
+
+    verifyComputedStyle("width", "calc(20.875 * 8lh + 7px)", "3347px", "testing width: calc(20.875 * 8lh + 7px)");
+    /*
+      8lh = 160px
+      20.875 * 160px + 7px = 3340px + 7px = 3347px
+      Total = 3347px
+    */
+
+    verifyComputedStyle("height", "calc(10% * 5.8 + 6pt)", "298px", "testing height: calc(10% * 5.8 + 6pt)");
+    /*
+      10% of 500px = 50px
+      6pt = 8px
+      50px * 5.8 + 8px = 290px + 8px = 298px
+      Total = 298px
+    */
+
+    verifyComputedStyle("width", "calc(2 * 4em + 2.6458333cm)", "228px", "testing width: calc(2 * 4em + 2.6458333cm)");
+    /*
+      4em = 64px
+      2.6458333cm = 100px
```

Comment:
Please avoid periodic decimals. `2.6458333cm` is (mathematically) slightly less than 100px, and who knows what browsers will do with their different precisions.

1cm is `96px/2.54`, so I would just use `calc(2 * 4em + 2.54cm)` and expect `224px`.

Reply:
Just fyi I am noticing lots of tests across the repo with that same decimal paradigm:

<img width="384" alt="" src="https://github.com/web-platform-tests/wpt/assets/5913254/1f6a3b35-c385-4623-abc1-8db9de7a2b26">

Reply:
But are they using `2.6458333cm`?

Reply:
The one file I copied most of this from initially did, but only that one
<img width="530" alt="" src="https://github.com/web-platform-tests/wpt/assets/5913254/c641e45f-8b2e-4bf6-afd3-0a2753dd7d14">

Reply:
Well, if it's an existing test then no problem, I was just concerned that it would be a problem to add it into a new test.

Reply:
So we can just leave as is then?

Reply:
Yes, if the old test is not problematic, no need to change it.